.teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt

```diff
@@ -17,6 +18,21 @@ class CIConfigIntegrationTests {
     }
 
     @Test
+    fun macOSBuildsSubset() {
+        val m = CIBuildModel()
+        val p = RootProject(m)
+        val releaseAccept = p.subProjects.find { it.name.contains("Release Accept") }
+        val macOS = releaseAccept?.subProjects?.find { it.name.contains("Macos") }
```

Comment:
Please make `releaseAccept` and `macOS` non-null, by appending `!!` to the expressions. Then we can get rid of all the `?`.

Reply:
done